Commit 7ca34e0baa7fa65efc929eee5b6b0c3d2104db8e already reverted one
change that caused the cwd of all apps spawned by krunner to
erroneously be set to the root dir.

That regressions is back so it's more robust to fix in within krunner.

Signed-off-by: Andrea Arcangeli <aarcange@redhat.com>
---
 krunner/main.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/krunner/main.cpp b/krunner/main.cpp
index 4593687ca..fa4a62ac4 100644
--- a/krunner/main.cpp
+++ b/krunner/main.cpp
@@ -12,6 +12,7 @@
 #include <QDBusConnection>
 #include <QDBusMessage>
 #include <QDebug>
+#include <QDir>
 #include <QQuickWindow>
 #include <QSessionManager>
 #include <QUrl>
@@ -112,5 +113,6 @@ int main(int argc, char **argv)
         }
     });
 
+    QDir::setCurrent(QDir::homePath());
     return app.exec();
 }

